pkg/daemon/daemon.go

```diff
@@ -1190,6 +1197,21 @@ func (dn *Daemon) onConfigDrift(err error) {
 	}
 }
 
+// substituteMissingODC fetch the current config through node annotations to respond to getCurrentConfigDisk
+// calls where the ODC is missing due to manual deletion and other reasons.
+func (dn *Daemon) substituteMissingODC() (*onDiskConfig, error) {
```

Comment:
I think instead of `substituteMissingODC` you can just call this `getCurrentConfigFromNode` or similar, to make it more descriptive of the functionality